sdk/securitycenter/Azure.ResourceManager.SecurityCenter/src/autorest.md

```diff
@@ -0,0 +1,107 @@
+  - /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/locations/{ascLocation}/allowedConnections/{connectionType}
+  - /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/locations/{ascLocation}/topologies/{topologyResourceName}
+  - /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/locations/{ascLocation}/discoveredSecuritySolutions/{discoveredSecuritySolutionName}
+  - /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/locations/{ascLocation}/ExternalSecuritySolutions/{externalSecuritySolutionsName}
+  - /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/locations/{ascLocation}/securitySolutions/{securitySolutionName}
+
+directive:
+  - rename-operation:
+      from: SecurityConnectorApplication_Get
+      to: SecurityConnectorApplications_Get
+  - rename-operation:
+      from: SecurityConnectorApplication_CreateOrUpdate
+      to: SecurityConnectorApplications_CreateOrUpdate
+  - rename-operation:
+      from: SecurityConnectorApplication_Delete
+      to: SecurityConnectorApplications_Delete
+  - rename-operation:
+      from: SecurityConnectorGovernanceRule_List
+      to: SecurityConnectorGovernanceRules_List
+  - from: externalSecuritySolutions.json
+    where: $.definitions
+    transform: >
+      $.ExternalSecuritySolutionKind.properties.kind['x-ms-enum']['name'] = 'ExternalSecuritySolutionKindType';
+      $.AadConnectivityState.properties.connectivityState['x-ms-enum']['name'] = 'AadConnectivityStateType';
+  - from: jitNetworkAccessPolicies.json
+    where: $.definitions
+    transform: >
+      $.JitNetworkAccessPortRule.properties.maxRequestAccessDuration['format'] = 'duration';
+  - from: types.json
+    where: $.definitions.Kind
+    transform: >
+      $['x-ms-client-name'] = 'ResourceKind';
+  - from: securityConnectors.json
+    where: $.definitions
+    transform: >
+      $.defenderForServersAwsOffering.properties.vaAutoProvisioning.properties.configuration.properties.type['x-ms-enum']['name'] = 'VAAutoProvisioningType';
+      $.defenderForServersAwsOffering.properties.subPlan.properties.type['x-ms-enum']['name'] = 'AvailableSubPlanType';
+      $.defenderForServersGcpOffering.properties.vaAutoProvisioning.properties.configuration.properties.type['x-ms-enum']['name'] = 'VAAutoProvisioningType';
+      $.defenderForServersGcpOffering.properties.subPlan.properties.type['x-ms-enum']['name'] = 'AvailableSubPlanType';
+  - from: alerts.json
+    where: $.definitions
+    transform: >
+      $.ResourceIdentifier['x-ms-client-name'] = 'AlertResourceIdentifier';
+  # TODO: temporary remove these operations to mitigate the exception from BuildParameterMapping in Autorext.CSharp
+  - remove-operation: InformationProtectionPolicies_Get
```

Comment:
Can we create an issue to track this problem?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/30769